vm-allocator/src/gsi.rs

```diff
@@ -29,12 +33,64 @@ impl GsiApic {
     }
 }
 
+#[cfg(target_arch = "x86_64")]
+pub struct GsiBitmap {
```

Comment:
Why don't you use the bitmap provided by vm-memory crate?

Reply:
The Bitmap in vm-memory seems a bit heavy in our this scenario. All we want is a bitmap which only has enable/disable one bit function.
Also, our bitmap code is very tiny which has only 50 lines :)

Reply:
It's not too heavy as we have used it before in MSHV hypervisor implementations. There are multiple implementations available. Please take a look if that does not work for you, feel free to add. But adding more code is also a maintenance burden,

Reply:
Yes, I agree. But instead of adding more Bitmap related code, I would prefer using/enhancing exiting ones.

Reply:
> https://crates.io/crates/bit-vec seems to be the popular choice.

Thx, I will look into that.

Reply:
Ahhhh, sorry, my fault. I set the bitmap incorrectly when change GsiBitmap to bit-vec, which I thought is a simple change and didn't check carefully.
Now it seems work fine.